nf_core/launch.py

```diff
@@ -347,14 +337,14 @@ def sanitise_web_response(self):
         The web builder returns everything as strings.
         Use the functions defined in the cli wizard to convert to the correct types.
         """
-        # Collect pyinquirer objects for each defined input_param
-        pyinquirer_objects = {}
+        # Collect questionary objects for each defined input_param
+        questionare_objects = {}
```

Comment:
Typo - plus additional references to same variable below.

```suggestion
        questionary_objects = {}
```